DOCS-2305: Email corrections (#1306)
MultiSafepay/docs: content/tools/credit-card-component/_index.md

```
### Contact

> If you have any questions regarding the process or face any issues, contact our Integration Team at <integration@multisafepay.com>

Reviewer: ```suggestion
For any questions or support, email the Integration Team at <integration@multisafepay.com>
```
